pom.xml

```diff
@@ -5,7 +5,7 @@
     <name>Cloudburst API</name>
     <groupId>org.cloudburstmc</groupId>
     <artifactId>cloudburst-api</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>0.0.3-SNAPSHOT</version>
```

Comment:
just a quick note - bumping the patch is fine but I recommend not bumping the major or minor versions until the API is actually properly usable

Reply:
I agree. Until we get this fully working with the server code, it's not worth bumping the version.

Reply:
Updated.